lib/modules/manager/npm/extract/common/package-file.ts

```diff
@@ -82,6 +83,28 @@ export function extractPackageJson(
                 val as unknown as NpmManagerData,
               ),
             );
+          } else if (depType === 'pnpm' && depName === 'overrides') {
+            for (const [overridesKey, overridesVal] of Object.entries(
+              val as unknown as NpmPackageDependency,
+            )) {
+              if (is.string(overridesVal)) {
+                dep = {
+                  depName: overridesKey,
+                  depType: 'overrides',
+                  ...extractDependency(depName, overridesKey, overridesVal),
+                };
+                setNodeCommitTopic(dep);
+                dep.prettyDepType = depTypes[depName];
+                deps.push(dep);
+              } else {
```

Comment:
can you validate that it's object here? and not a bool or anything else because of a user error?

Reply:
added in the last commit!